src/preset_cli/cli/superset/lib.py

```diff
@@ -0,0 +1,77 @@
+"""
+Helper functions for the Superset commands
+"""
+
+from __future__ import annotations
+
+from pathlib import Path
+from typing import Any, Dict, Set
+
+import yaml
+
+LOG_FILE_PATH = Path("progress.log")
+
+
+def get_logs() -> dict[str, Any]:
```

Comment:
Yeah, let's create a `LogType` (or something) and return it here, with owners and assets set to empty lists if not present.

Reply:
Covered here https://github.com/preset-io/backend-sdk/pull/335